extensions/namespace-lookup/src/test/java/io/druid/server/namespace/cache/JDBCExtractionNamespaceTest.java

```diff
@@ -196,7 +196,7 @@ public Void withHandle(Handle handle) throws Exception
     );
   }
 
-  @Test(timeout = 500)
+  @Test(timeout = 50_000)
```

Comment:
For long-running test we typically default to 60s timeout (see ChainedExecutionQueryRunner for instance). I think it makes sense to use the same value everywhere. We should probably make that a constant at some point.